translations/README.de.md

```diff
@@ -0,0 +1,74 @@
+# GitHub Resume Generator
+
+![React GitHub Resume](../src/assets/readme/screenshot.png)
+
+## Inhalt
+
+- [Über das Projekt](#about)
+- [Werkzeuge](#built-with)
+- [Installation](#installation)
+- [Demo](#live-demo)
+
+### 🌎 _Lies in [deiner Sprache](.Translations.md) weiter_
+
+<h2 id='about'>🤔 Über das Projekt</h2>
+
+Ziel dieses Projektes ist es einen `Github Resume` für Github User zu erzeugen.
+
+<h2 id='built-with'>🛠️ Werkzeuge</h2>
+
+- [React JS](https://reactjs.org/) - Front-End JavaScript Bibliothek
+- [Material UI](https://material-ui.com/) - React UI Framework
+
+<h2 id='installation'>Installation</h2>
+
+Um die App zur Entwicklung lokal zu installieren folge bitte folgenden Anweisungen:
+
+1. Klone diese Repository auf deinen Rechner und wechsle in das Verzeichnis
+
+   ```bash
+   git clone https://github.com/sabesansathananthan/React-GitHub-Resume.git
+   cd React-GitHub-Resume
+   ```
+
+2. Paketinstallation
+
+   Solltest Du `npm` nutzen:
+
+   ```bash
+   npm install
+   ```
+
+   oder
+
+   wenn Du `yarn` benutzt:
+
+   ```bash
+   yarn
+   ```
+
+3. Starten des Development Server
+
+   Solltest Du `npm` benuzten:
+
+   ```bash
+   npm start
+   ```
+
+   oder
+
+   wenn Du `yarn` nutzt:
+
+   ```bash
+   yarn start
+   ```
+
+4. Öffne <http://localhost:3000> in deinem Browser
+
+<h2 id='live-demo'><a href="https://react-github-resume.vercel.app/">Live Demo</a></h2>
```

Comment:
Is Live Demo is same as Live Demo in German

Reply:
yes - its not common to call it something else